features/functests/performance/performance.go

```diff
@@ -29,7 +29,7 @@ var _ = Describe("performance", func() {
 	var _ = Context("RT Kernel Arguments", func() {
 		It("Nodes should contain kernel arguments set by machine configuration", func() {
 			nodes, err := clients.K8s.CoreV1().Nodes().List(metav1.ListOptions{
-				LabelSelector: "node-role.kubernetes.io/worker=",
+				LabelSelector: "node-role.kubernetes.io/worker-rt=",
```

Comment:
Can you also assert that len(nodes) > 0? Not sure if `List` returns an error if the list is empty